talent360/evaluaciones/models/usuario_evaluacion_rel.py

```diff
@@ -127,7 +128,6 @@ def enviar_evaluacion_action(self, evaluacion_id):
                     correo = usuario.usuario_externo_id.email
                     nombre = usuario.usuario_externo_id.nombre
                 else:
-                    logging.error("No se encontró un usuario asociado")
                     raise ValueError("No se encontró un usuario asociado")
```

Comment:
Please check talent360/evaluaciones/models/usuario_evaluacion_rel.py for pylint errors